c7n/resources/account.py

```diff
@@ -532,12 +533,15 @@ def process(self, resources):
         limit_exceeded = resources[0].get('c7n:ServiceLimitsExceeded', [])
         percent_increase = self.data.get('percent-increase')
         amount_increase = self.data.get('amount-increase')
-
+        minimum_increase = self.data.get('minimum-increase')
```

Comment:
rather than doing a check for if minimum-increase is None, you can just specify the default in the .get function itself, i.e. self.data.get('minimum-increase', 1)

Reply:
aha, indeed. Also, spelling counts, apparently ;-)